<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostDiaDigit</class>
 <widget class="QMainWindow" name="PostDiaDigit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">PostDiaDigit 0.90</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>501</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Allgemeines</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>451</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>File Formate</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Primär:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_ext1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>jpg</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sekundär:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_ext2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>nef</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Sprache</string>
     </property>
     <widget class="QComboBox" name="cb_lang">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string notr="true">Deutsch</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string notr="true">English</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string notr="true">Français</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>609</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionEinstellungen_speichern"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuBearbeiten">
    <property name="title">
     <string>Bearbeiten</string>
    </property>
    <addaction name="actionUmbenennen"/>
    <addaction name="actionSchneiden"/>
    <addaction name="actionDubletten"/>
    <addaction name="actionRahmen"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionHilfe"/>
    <addaction name="separator"/>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuBearbeiten"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEinstellungen_speichern">
   <property name="text">
    <string>Einstellungen speichern</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionUmbenennen">
   <property name="text">
    <string>Umbenennen</string>
   </property>
  </action>
  <action name="actionSchneiden">
   <property name="text">
    <string>Schneiden</string>
   </property>
  </action>
  <action name="actionDubletten">
   <property name="text">
    <string>Dubletten</string>
   </property>
  </action>
  <action name="actionRahmen">
   <property name="text">
    <string>Rahmen</string>
   </property>
  </action>
  <action name="actionHilfe">
   <property name="text">
    <string>Hilfe</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
